src/env/index.ts

```diff
@@ -1,7 +1,5 @@
 import Config from 'react-native-config';
 
-export const APP_ID = Config.APP_ID;
```

Comment:
NIT: I would prefer to keep it consistent by adding all fields in `.env` to ts layer.

Reply:
that would mean we expose both iOS and Android configs _at the same time_ which doesn't make sense to me. Aldo, we never use APP_ID in the codebase, so I figured, if ever need to — we'll do it properly, by using platform selectors.

Reply:
I pushed a fix.